qiskit_ibm_provider/api/clients/account.py

```diff
@@ -31,7 +31,6 @@
     UserTimeoutExceededError,
 )
 from ..rest import Api, Account
-from ..rest.backend import Backend
 from ..session import RetrySession
 from ...utils.hgp import from_instance_format
 
```

Comment:
In theory this entire module can go away, right? Since it's for using IQX API.

Reply:
I'm still using it as a backend adapter for the updated backend related endpoints, so I think it can stay

Reply:
I don't understand, where are you still using `AccountClient`? Isn't it only used for h/g/p specific endpoints?